src/bridge-deployer/src/config/rune.rs

```diff
@@ -92,6 +92,7 @@ impl From<RuneBridgeConfig> for rune_bridge::state::RuneBridgeConfig {
             indexer_urls: value.indexer_urls.into_iter().collect(),
             deposit_fee: value.deposit_fee,
             mempool_timeout: Duration::from_secs(value.mempool_timeout),
+            indexer_consensus_threshold: value.no_of_indexers,
```

Comment:
Threshold should be another argument, because we could have 5 indexers, but a threshold of 4 for instance

Reply:
This is parameters of the bridge deploy tool. I don't think we need a separate parameter in CLI for this.